source/getting-started/installation-raspberry-pi-all-in-one.markdown

```diff
@@ -49,7 +49,7 @@ The All-In-One Installer script will do the following automatically:
 *  Install Home Assistant in a virtualenv
 *  Install Mosquitto with websocket support running on ports 1883 and 9001
 *  Build and Install Python-openzwave in the Home Assistant virtualenv
-*  Build openzwave-control-panel in `/srv/homeassistant/src/open-zwave-control-panel`
+*  Build openzwave-control-panel in `/srv/hass/src/open-zwave-control-panel`
```

Comment:
Are you sure that this change is accurate? See https://home-assistant.io/getting-started/installation-raspberry-pi-all-in-one/